core/local/watcher.js

```diff
@@ -132,7 +133,8 @@ module.exports = class LocalWatcher {
     const started = new Promise((resolve) => {
       for (let eventType of ['add', 'addDir', 'change', 'unlink', 'unlinkDir']) {
         this.watcher.on(eventType, (path /*: ?string */, stats /*: ?fs.Stats */) => {
-          log.chokidar.debug({path, stats}, eventType)
+          const initialScan = this.initialScan && this.initialScan.flushed
```

Comment:
Isn't the condition reversed?

```js
const initialScan = this.initialScan && !this.initialScan.flushed
```

Also, out of the scope of this PR, but what about `const isInitialScan` to prevent confusion with the actual object (my bad)?